shared/components/ArchivedCommunityEvent/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react';
+
+import { ExternalLink } from '../ExternalLink';
+import { Link } from 'react-router';
+import Talks from '../Talks';
+import { formatDate } from '../../utilities/date';
+import locationType from '../../prop-types/location-type';
+
+export const placeholderText = 'To be confirmed.';
+
+function eventDateAndTime(startDateTime, endDateTime) {
+  if (startDateTime && startDateTime.iso && endDateTime && endDateTime.iso) {
+    return (
+      formatDate(startDateTime, 'dddd, Do MMMM YYYY, HH:mm – ') +
+      formatDate(endDateTime, 'HH:mm')
+    );
+  }
+  return placeholderText;
+}
+
+class ArchivedCommunityEvent extends React.Component {
+  renderVideoButton() {
+    const featuredEvent = this.props;
+    const { streamingLink } = featuredEvent;
```

Comment:
no point having both, you only use `featuredEvent` for it's `eventId` below so this could just be:
```js
const { eventId, streamingLink } = this.props.featuredEvent;
```